controller/search_blackbox_test.go

```diff
@@ -942,3 +942,84 @@ func (s *searchBlackBoxTest) TestIncludedParents() {
 	}
 	assert.Equal(s.T(), successCnt, 4)
 }
+
+func (s *searchBlackBoxTest) TestUpdateWorkItem() {
+	resetFn := s.DisableGormCallbacks()
+	defer resetFn()
+
+	s.T().Run("assignees", func(t *testing.T) {
+		t.Run("filter null", func(t *testing.T) {
+			fxt := tf.NewTestFixture(t, s.DB, tf.CreateWorkItemEnvironment(), tf.WorkItems(1))
+			spaceInstance := fxt.Spaces[0]
+			spaceIDStr := spaceInstance.ID.String()
+			filter := fmt.Sprintf(`
+							{"assignee":null}`,
+			)
+			_, result := test.ShowSearchOK(t, nil, nil, s.controller, &filter, nil, nil, nil, nil, &spaceIDStr)
+			wi := result.Data[0]
+			t.Run("update update item", func(t *testing.T) {
```

Comment:
`update update item` looks wrong.

Reply:
Hmm.. I am not sure what else can be given here.